### PR TITLE
Release/2.2.0

### PR DESCRIPTION
**Pull request by repjarms**
### Summary

This PR introduces a few features and some fixes for what will be version 2.2.0 of the SDK.

Assistant V1
alt_text property added to Image response type
​
Assistant V2
session_start_time and state properties added to MessageContextGlobalSystem model
alt_text property added to Image response type
location property of RuntimeEntity model changed from required to optional
​
Discovery V2
Enum update for CreateProjectConstants
​
NLU
Fix for listClassificationsModels through return type change and removal of unnecessary model
​
STT
More languages supported for next generation models
​
TTS
New voice models added
​
CC, Disco V1, LT, NLC, PI, TA, VisRec V3 & V4
No changes

### CODE REVIEW

**Comment by repjarms** [comment]
@nan2iz @apaparazzi0329 this PR is ready for review and able to be merged upon approval. Unit tests have been run locally and on remote, and integration tests have been run locally.

File: v2/discoveryv1/discovery_v1.go [hunk 1]
```diff
@@ -4722,10 +4722,7 @@ func UnmarshalAggregationResult(m map[string]json.RawMessage, result interface{}
 	obj := new(AggregationResult)
 	err = core.UnmarshalPrimitive(m, "key", &obj.Key)
 	if err != nil {
-		err = core.UnmarshalPrimitive(m, "key_as_string", &obj.Key)
-		if err != nil {
-			return
-		}
```
Comment (nan2iz) [inline v2/discoveryv1/discovery_v1.go:4728]: oh just curious why we remove this part out?

**Review by nan2iz — DISMISSED** [review]
Looks good to me

File: v2/discoveryv1/discovery_v1.go (hunk 1 above)
Comment (repjarms) [reply v2/discoveryv1/discovery_v1.go:4728]: That's a great catch. That needs to be re-added as a hand edit.

**Comment by repjarms** [comment]
WIP

File: v2/discoveryv1/discovery_v1.go (hunk 1 above)
Comment (nan2iz) [reply v2/discoveryv1/discovery_v1.go:4728]: Okay. I will update the review to `request changes`

**Review by nan2iz — CHANGES_REQUESTED** [review]
As dicussed. Switch to review to `Request changes`

Until this part is added back to `discovery_v1` on line 4725
```
err = core.UnmarshalPrimitive(m, "key_as_string", &obj.Key)
		if err != nil {
			return
		}
```

**Comment by repjarms** [comment]
@nan2iz @apaparazzi0329 PR is updated and ready for re-review

File: v2/discoveryv1/discovery_v1_suite_test.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018, 2021.
+ * (C) Copyright IBM Corp. 2021.
```
Comment (nan2iz) [inline v2/discoveryv1/discovery_v1_suite_test.go:2]: oh I wonder if you need to run lint on the copyright again?

**Review by nan2iz — APPROVED** [review]
Overall looks good to me. But I guess you may need to rerun linter again for the copyrights?

File: v2/discoveryv1/discovery_v1_suite_test.go (hunk 2 above)
Comment (repjarms) [reply v2/discoveryv1/discovery_v1_suite_test.go]: @nan2iz yep good catch. I just updated them.

**Review by nan2iz — APPROVED** [review]
Looks good to me!

**Pull request merged**